cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -3,143 +3,31 @@
    Bureau structure page at /the-bureau/bureau-structure/.
    ========================================================================== */
 
-.org-chart {
 
-    .respond-to-min(@bp-sm-min, {
-        width: 90%;
-        margin: 0 auto;
-    });
+@grid_gutter-width__em: unit(@grid_gutter-width / @base-font-size-px, em);
+@spine-width__px: 5px;
+@spine-color: #BABBBD;
```

Comment:
Got some colors for this and the other colors below from @ajbush.

![image](https://cloud.githubusercontent.com/assets/1860176/13678799/95b00fa0-e6be-11e5-9c64-e867ceb398f3.png)